torch/fx/passes/graph_drawer.py

```diff
@@ -273,75 +277,60 @@ def _get_str_for_args_kwargs(arg):
             # print buf=buf0, n_origin=6
             buf_meta = node.meta.get('buf_meta', None)
             if buf_meta is not None:
-                label += f"|buf={buf_meta.name}" + r"\n"
-                label += f"|n_origin={buf_meta.n_origin}" + r"\n"
+
+                label += f'<tr><td>buf</td><td>{buf_meta.name}</td></tr>'
+                label += f'<tr><td>n_origin</td><td>{buf_meta.n_origin}</td></tr>'
 
             # for original fx graph
             # print file:lineno code
             if parse_stack_trace and node.stack_trace is not None:
                 parsed_stack_trace = _parse_stack_trace(node.stack_trace)
                 fname = self._shorten_file_name(parsed_stack_trace.file)
-                label += f"|file={fname}:{parsed_stack_trace.lineno} {parsed_stack_trace.code}" + r"\n"
-
+                label += f'<tr><td>file</td><td>{fname}:{parsed_stack_trace.lineno} {parsed_stack_trace.code}</td></tr>'
 
-            return label + "}"
+            return label + '</table>>'
 
         def _tensor_meta_to_label(self, tm) -> str:
             if tm is None:
                 return ""
             elif isinstance(tm, TensorMetadata):
                 return self._stringify_tensor_meta(tm)
-            elif isinstance(tm, list):
-                result = ""
-                for item in tm:
-                    result += self._tensor_meta_to_label(item)
-                return result
+            elif isinstance(tm, (list, tuple)):
+                return "".join(self._tensor_meta_to_label(item) for item in tm)
             elif isinstance(tm, dict):
-                result = ""
-                for v in tm.values():
-                    result += self._tensor_meta_to_label(v)
-                return result
-            elif isinstance(tm, tuple):
-                result = ""
-                for item in tm:
-                    result += self._tensor_meta_to_label(item)
-                return result
+                return "".join(self._tensor_meta_to_label(v) for v in tm.values())
             else:
                 raise RuntimeError(f"Unsupported tensor meta type {type(tm)}")
 
         def _stringify_tensor_meta(self, tm: TensorMetadata) -> str:
```

Comment:
Same as below, would be nice to change func name to indicate it returns an html-like string now
```suggestion
        def _htmlize_tensor_meta(self, tm: TensorMetadata) -> str:
```